plugin/http/httptrace/httptrace_test.go

```diff
@@ -23,6 +23,14 @@ import (
 	"testing"
 
 	"go.opencensus.io/trace"
+	"go.opencensus.io/trace/propagation"
+	"strings"
```

Comment:
Minor nit, could we move the stdlib imports to make them part of the imports in line 23 and keep the blank line for the ones for `"go.opencensus.io/*"` in their own small block.